### PR TITLE
31042 navbar styling

### PR DESCRIPTION
**Pull request by bienvu**
Add superfish menu to component.

### CODE REVIEW

**Comment by joannguyen** [comment]
you can review code now

File: themes/agov/agov_whitlam/sass/style-guide/kss-only.scss [hunk 1]
```diff
@@ -15,3 +15,14 @@
...
+.demo-bg {
+	padding-top: 60px;
+  background: color(link);
+  position: relative;
+
+	@include respond-to('medium') {
+    background: none;
+    padding: 0;
+  }
+}
```
Comment (kimpepper) [inline themes/agov/agov_whitlam/sass/style-guide/kss-only.scss]: Needs newline at end of file.

File: themes/agov/agov_whitlam/js/jquery.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*! jQuery v1.10.1 | (c) 2005, 2013 jQuery Foundation, Inc. | jquery.org/license
```
Comment (kimpepper) [inline themes/agov/agov_whitlam/js/jquery.js]: This will break panels IPE. Need to use core Drupal jquery

Comment (bienvu) [inline themes/agov/agov_whitlam/js/jquery.js]: Drupal jquery does not work in styleguide page.

File: themes/agov/agov_whitlam/sass/style-guide/kss-only.scss (hunk 1 above)
Comment (bienvu) [inline themes/agov/agov_whitlam/sass/style-guide/kss-only.scss]: fixed

**Comment by kimpepper** [comment]
Is this meant to include the mobile nav?

**Comment by HierenFeb** [comment]
yes. including the mobile nav

**Comment by kimpepper** [comment]
The nav disappears for me
nav-desktop
nav-mobile

 when I resize it to mobile width.

**Comment by kimpepper** [comment]
@HierenFeb said this can be fixed once PRs are merged

**Pull request merged**